### PR TITLE
Shuttle Wall visualizer fix

### PR DESCRIPTION
**Pull request by IamVelcroboy**
Changelog:

- Fixes visual bug when deconstructing regular shuttle walls 
- Adds suffix for better mapping clarity

### CODE REVIEW

File: Content.Client/Wall/ReinforcedShuttleWallVisualizer.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+using Robust.Client.GameObjects;
+using Robust.Shared.GameObjects;
+using Robust.Shared.IoC;
+
+namespace Content.Client.Wall
+{
+    [UsedImplicitly]
+    public sealed class ReinforcedShuttleWallVisualizer : AppearanceVisualizer
+    {
+        [Obsolete("Subscribe to AppearanceChangeEvent instead.")]
```
Comment (Elijahrane) [inline Content.Client/Wall/ReinforcedShuttleWallVisualizer.cs:12]: You're adding new code here that says it's already obsolete...

Comment (IamVelcroboy) [reply Content.Client/Wall/ReinforcedShuttleWallVisualizer.cs:12]: I saw that but it's what's currently being used for the other walls.... So, I wanted to keep it consistent.

Should I change everything over to the new one?

**Comment by IamVelcroboy** [comment]
So yeah, I;m not even finding anything related to this "AppearanceChangeEvent". So, I'm not really sure how to proceed. I _do_ know that this fixes the issue that is currently present. So, even if a temporary fix, I feel like it should be implemented. Unless someone can point me in the right direction to do it another way?

**Comment by IamVelcroboy** [comment]
Alternatively, if someone else more familiar with the code base wants to take the sprites from here and fix it, that would be fine too.

**Comment by Vordenburg** [comment]
I'll have to take a thorough look into this.

**Comment by IamVelcroboy** [comment]
OK, here we go [15735](https://github.com/space-wizards/space-station-14/pull/15735/files). I'll take a look at this next chance I get.

**Comment by Vordenburg** [comment]
@ChilbroBaggins AppearanceVisualizers were completely removed in the most recent update. Is your fix still necessary, or will it need some alteration?

**Comment by IamVelcroboy** [comment]
> @ChilbroBaggins AppearanceVisualizers were completely removed in the most recent update. Is your fix still necessary, or will it need some alteration?

It will need to mirror whatever the process for deconstructing r-walls uses. I'll take a look soon. I should be able to fix it.

**Comment by IamVelcroboy** [comment]
Reopening this with proper fix

**Pull request closed without merging**